lib/elixir/lib/module.ex

```diff
@@ -590,6 +591,40 @@ defmodule Module do
       `@compile {:no_warn_undefined, {Mod, fun, arity}}` - does not warn if
       the given module or the given `Mod.fun/arity` are not defined
 
+  ## Generated Functions
+
+  Sometimes the compiler will generated public functions within modules. These
+  are documented below.
```

Comment:
Should we document them `module_info` generated by Erlang `__info__` generated by Elixir?

Reply:
Yep, added!